package.nls.json

```diff
@@ -1,6 +1,7 @@
 {
     "staticWebApps.createStaticWebApp": "Create Static Web App...",
     "staticWebApps.deleteStaticWebApp": "Delete...",
+    "staticWebApps.deletedeleteEnvironment": "Delete Environment...",
```

Comment:
deletedeleteEnvironment -> deleteEnvironment

Reply:
What the heck, I remember fixing this...